src/Component/JsonSchema/Guesser/JsonSchema/AllOfGuesser.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Jane\Component\JsonSchema\Guesser\JsonSchema;
 
+use Jane\Component\AutoMapper\Exception\RuntimeException;
```

Comment:
Is it ok to use this, or is it better to add a custom one? Or the standard PHP one?

Reply:
Add a custom one please, we can't use something from another component that is not required in the component you are.